src/contained.js

```diff
@@ -0,0 +1,28 @@
+import { flip, contains } from 'ramda';
+
+/**
+ * Returns true if the specified value is equal, in R.equals terms,
+ * to at least one element of the given list or false otherwise.
+ * Given list can be a string.
+ *
+ * Like {@link http://ramdajs.com/docs/#contains|R.contains} but with argument order reversed.
+ *
+ * @func contained
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.7.0|v2.7.0}
+ * @category List
+ * @sig [a] → a → Boolean
+ * @param {Array|String} list The list to consider
+ * @param {*} a The item to compare against
+ * @return {Boolean} Returns Boolean `true` if an equivalent item is in the list or `false` otherwise.
+ * @see {@link http://ramdajs.com/docs/#contains|R.contains}, {@link http://ramdajs.com/docs/#flip|R.flip}
```

Comment:
Don't think we need the link to `R.flip` as that is an implementation detail.

Reply:
I don't understand, it's already done.

Reply:
I'm suggesting you *remove* the reference to `R.flip`.

Reply:
it's done.